<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgStoreUtils</class>
 <widget class="QDialog" name="DlgStoreUtils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbDocsCount">
     <property name="text">
      <string>DocsCount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbCurrentlyProceed">
     <property name="text">
      <string>Current</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbError">
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnWriteAllDocuments">
     <property name="text">
      <string>Write All Documents</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
